all/src/assemble/NOTICE.bin.txt

```diff
@@ -94,13 +94,82 @@ Copyright 2004 Jason Paul Kitchen
 TypeUtil.java
 Copyright 2002-2012 Ramnivas Laddad, Juergen Hoeller, Chris Beams
 
+picocli (http://picocli.info)
+Copyright 2017 Remko Popma
+
 ----------------------------------------------------------------------------------------------------
 
 Prometheus instrumentation library for JVM applications
 Copyright 2012-2015 The Prometheus Authors
 
+This product includes software developed at
+Boxever Ltd. (http://www.boxever.com/).
+
+This product includes software developed at
+SoundCloud Ltd. (http://soundcloud.com/).
+
+This product includes software developed as part of the
+Ocelli project by Netflix Inc. (https://github.com/Netflix/ocelli/).
```

Comment:
just wanted to check which binary belong to this one,?

Reply:
I see.. so, we have to mention license/notice for transitive dependencies as well?

Reply:
That we are always doing. Here is reporting in our NOTICE file what is in the NOTICE file from our dependencies (both first level and transitives). In particular this refers to libraries/code that is bundled inside a particular dependency.